stage0/run.go

```diff
@@ -452,6 +457,8 @@ func Run(cfg RunConfig, dir string, dataDir string) {
 		log.Fatalf("error clearing FD_CLOEXEC on lock fd")
 	}
 
+	tpmEvent := fmt.Sprintf("rkt: Rootfs: %s Manifest: %s Stage 1 args: %s", cfg.CommonConfig.RootHash, cfg.CommonConfig.ManifestData, strings.Join(args, " "))
+	tpm.Extend(tpmEvent)
```

Comment:
Right now we should just continue anyway. I've added a comment to that effect. Longer term we'll want the ability to enforce TPM use and so failures here should be fatal, but that's going to involve a more global security policy.